src/js/src/mixins/HistoryRoutingUtils.js

```diff
@@ -24,6 +24,17 @@ export default {
         currentParams.facets !== appliedFacets.join('') ||
         currentParams.offset !== solrSettings.offset ||
         currentParams.sort !== solrSettings.sort)
+    },
+    $_getResolvedUrl(destination, query, appliedFacets, solrSettings) {
+      const route = this.$router.resolve({name: destination, query: { query: query,
```

Comment:
So good practice you utilised $router.resolve here.

Reply:
:+1: